modules/TestUtils.js

```diff
@@ -20,6 +20,12 @@ export const isObject = (object) =>
   object && !isArray(object) && typeof object === 'object'
 
 /**
+ * Returns true if the given object is an empty object.
+ */
+export const isEmptyObject = (object) =>
+  isObject(object) && Reflect.ownKeys(object).length === 0
```

Comment:
@ljharb  The new tests use `Symbol()`, so they can not pass in `node@0.23`. Should I add a polyfill for it or just remove those tests?